docs/semantic-search.ipynb

```diff
@@ -55,333 +56,236 @@
       ]
     },
     {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "kujS_e8s55oJ"
-      },
-      "source": [
-        "In this notebook we will skip the data preparation steps as they can be very time consuming and jump straight into it with the prebuilt dataset from *Pinecone Datasets*.\n",
-        "\n",
-        "The dataset we are working with represents embeddings of [400K question pairs from Quora](https://quoradata.quora.com/First-Quora-Dataset-Release-Question-Pairs). The embeddings were created using the `all-MiniLM-L6-v2` model from Hugging Face via the `sentence-transformers` package.\n",
-        "\n",
-        "If you'd rather see how it's all done, please refer to [this notebook](https://github.com/pinecone-io/examples/blob/master/learn/search/semantic-search/semantic-search.ipynb).\n",
-        "\n",
-        "Let's go ahead and download the dataset."
-      ]
+      "cell_type": "code",
```

Comment:
Is this empty cell intentional?

Reply:
I think I took care of this, but I probably did in accident!